### ISSUE TITLE
Webp support for thumbnails

### ISSUE THREAD

**Issue by vitormmatos** [xiph/ogg#79]
Webp is currently a more compact format than jpeg when on lossy compression, same with PNG and lossless compression:
https://github.com/webmproject/libwebp/

Is it possible and viable to implement it for usage on thumbnails with the ogg container, either video or audio?

**Comment by rillian** [comment]
In Ogg metadata is attached to specific compressed formats, not the overall container.

It should work to put `image/web` in a `METADATA_BLOCK_PICTURE` tag in an Opus file, also the reference encoder app only has specific support for png/jpeg/gif.

**Comment by pitlochry** [comment]
Yes, it works, but not every player can make use of it, not even mplayer or mpv. But it could be implemented with ease.
